app/Main.hs

```diff
@@ -22,20 +23,24 @@ data CLI
       (Maybe FilePath) -- file to run
       Bool -- cheat mode
   | Format Input
+  | DocGen GenerateDocs
   | LSP
 
 cliParser :: Parser CLI
 cliParser =
   subparser
     ( command "format" (info (format <**> helper) (progDesc "Format a file"))
         <> command "lsp" (info (pure LSP) (progDesc "Start the LSP"))
+        <> command "generate" (info (docgen <**> helper) (progDesc "Generate docs"))
```

Comment:
I wonder if it is possible to have a subparser in a subparser, kind of like `docker image ps -a`, except in our case it would be `swarm generate recipes --graph`.

Reply:
After inspecting the types a bit, I found it. :grinning: